src/schemas/RowSchema.tsx

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+import {CellType, CommentType, Review_Stages} from '../components/TimeCardPage/types'; 
+
+const optionalNumber = z.union([z.undefined(), z.number()]) 
+
+const optionalMember = z.union([z.undefined(), z.object({
+    Start: optionalNumber, End: optionalNumber, Author: optionalNumber
+})]); 
+
+export const CommentSchema = z.object({
```

Comment:
I also have a `State` attribute for backend representation that is active or deleted. For now, can I plan to just default that state to active for all comments until we add in functionality for filtering comments/deleting comments?